### ISSUE TITLE
Possible issue with Generic.WhiteSpace.ScopeIndent.Incorrect using PSR2 

### ISSUE THREAD

**Issue by ikappas** [squizlabs/PHP_CodeSniffer#1345]
Using the PSR2 coding standard the following example shows only two errors where there should be six.
```php
<?php

echo "Test 0 spaces should show error";
  echo "Test 2 spaces should show error";
    echo "Test 4 spaces should pass";
       echo "Test 6 spaces should show error";

if ($case === 0) {
echo "Test 0 spaces should show error";
} elseif ($case === 2) {
  echo "Test 2 spaces should show error";
} elseif ($case === 4) {
    echo "Test 4 spaces should pass";
} elseif ($case === 6) {
       echo "Test 6 spaces should show error";
}

```

```bash
$ phpcs --standard=PSR2 -pns  example.php
E


FILE: ...some/path/example.php
----------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 2 LINES
----------------------------------------------------------------------
 10 | ERROR | [x] Line indented incorrectly; expected at least 4 spaces,
    |       |     found 0 (Generic.WhiteSpace.ScopeIndent.Incorrect)
 12 | ERROR | [x] Line indented incorrectly; expected at least 4 spaces,
    |       |     found 2 (Generic.WhiteSpace.ScopeIndent.Incorrect)
```

**Comment by gsherwood** [comment]
Lines need to be indented at least 4 spaces, but you can indent code more if you want. Control structures have more strict rules because they set the indent levels, but regular code lines don't actually need to line up.

If the sniff enforced this (which it can, by setting the [`exact` sniff property](https://github.com/squizlabs/PHP_CodeSniffer/wiki/Customisable-Sniff-Properties#genericwhitespacescopeindent)) it would mean you couldn't write code like:
```php
$foo = 'one'
    . 'two'
    . 'three';

// Or
$something
    ->foo()
    ->bar();

// Or
ob_start();
    echo 'foo';
    echo 'bar';
ob_end_flush();
```
None of the above code structures is banned in PSR2, but they don't use exact indenting.

Does that make sense?

**Comment by ikappas** [comment]
@gsherwood I see what you mean. So this is intended behaviour?

**Comment by gsherwood** [comment]
> So this is intended behaviour?

Yes, this is how the check is supposed to work.

**Comment by ikappas** [comment]
@gsherwood Thank you for your swift response!
